.github/workflows/xenial_build.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       OS_CODE_NAME: xenial
       ROS_DISTRO: kinetic
       ROS_REPO: main
-      TARGET_WORKSPACE: '. github:ros-industrial/cmake_common_scripts#master'
-      ROSDEP_SKIP_KEYS: "iwyu cmake_common_scripts"
+      UPSTREAM_WORKSPACE: 'dependencies.rosinstall'
+      ROSDEP_SKIP_KEYS: "iwyu ros_industrial_cmake_boilerplate"
```

Comment:
Is this still needed if `dependencies.rosinstall` lists it as something to add to the `UPSTREAM_WORKSPACE`?